app/assets/stylesheets/style.scss

```diff
@@ -81,3 +81,23 @@ body {
   position: fixed;
   width: 60px;
 }
+
+.post-title {
+  font-size: 125%;
+  font-weight: bold;
+}
+
+.post-body {
+  font-size: 90%;
+  font-weight: normal;
+  margin: 10px;
+}
+
+.course-buttons {
+  width:100%;
```

Comment:
Properties should be ordered text-align, width<br>Colon after property should be followed by one space